### PR TITLE
Add the tutorial on how to build an NFT marketplace from scratch using Solidity, IPFS, Hardhat, Ethers, Alchemy

### PR DESCRIPTION
**Pull request by ankg**
## Description

Developers often want comprehensive articles on how to build finished consumer products. One of the most common google searches for Developers building in NFT ecosystem is "How to build an NFT Marketplace". This article is written for programmers with all levels of experience. Even if you know only a little javascript or a little bit of coding, this article is super easy to follow.

This article walks through the following steps
1. Set up IPFS, Metamask and Alchemy
2. Write your solidity smart contract for an NFT marketplace. The contract has functions to create tokens, transfer, buy and sell.
3. Integrate your smart contract and other code with a pre-built frontend
4. Deploy smart contracts and also your application
5. Testing your NFT 

We have also built a Youtube video to cover audiences that finds articles hard to follow

### CODE REVIEW

**Comment by minimalsm** [comment]
Thanks for the PR :-) There are quite a few things you need to fix before we can review this. Could you please open a PR with the index.md edited [as per our style guide for tutorials.](https://ethereum.org/en/contributing/style-guide/)

Some examples:

- The images here are all broken
- Lots of random formatting characters that don't need to be here
- It starts at Step 7 currently
- Please add header ids to every header
- Frontmatter entirely missing except description
- Remove platform-specific functionality e.g. "{% tabs %}"
- For us to review this, you should be able to get this to build locally yourself and everything should look as it is supposed to 

Please refer to other tutorials for guidance on what is necessary or feel free to ping on Github or Discord.

Thanks, look forward to reading it 😀!

**Comment by ankg** [comment]
Comments are really helpful! Will fix these. Thanks @minimalsm !!
